subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/RealLifeAndroidStudioPerformanceTest.groovy

```diff
@@ -42,12 +44,7 @@ class RealLifeAndroidStudioMockupPerformanceTest extends AbstractCrossVersionPer
         runner.runs = 40
         runner.minimumBaseVersion = "6.5"
         runner.targetVersions = ["7.5-20211229231357+0000"]
-
-        runner.toolingApi("Android Studio Sync") {
-            it.action(new GetModel())
-        }.run { modelBuilder ->
-            SyncAction.withModelBuilder(modelBuilder)
```

Comment:
Since we are not using this any more, I think the whole `internal-android-performance-testing` subproject can be deleted, right?

Reply:
removed, let's see what happens :)